module/support/optimisation/NSGA2Evaluator/README.md

```diff
@@ -0,0 +1,25 @@
+NSGA2Evaluator
```

Comment:
Looks like this file might need updating?

Reply:
You should check out other readmes, eg BallDetector and follow the styling. 
The description sounds a bit specific to the walk engine scenario, but from what I understand this module is not tied to any particular tuning, ie it's modular and generic. So the description should be more generic. 
The usage might be good to mention how to actually set up for any tuning, but maybe that would be too long and better off in a NUbook page. A brief summary can go here though.
Emits shouldn't say where it goes to - everything here should be modular, it shouldn't matter who picks it up what matters is what's in the message. What does the module do to create the message? What is a FitnessScores? Why does it emit an ExecuteScript command? And also put the full scope for the message. MotionCommand is probably message::motion::WalkCommand right?
Add in a `Consumes` section. What messages does this module use?
Dependencies should be the libraries or maybe utilities as well that the module uses, not other NUClear reactors. You can specify sensors in the consumes section.